log.go

```diff
@@ -52,7 +52,7 @@ var (
 var subsystemLoggers = map[string]btclog.Logger{
 	"BTCW": log,
 	"WLLT": walletLog,
-	"TXST": txstLog,
+	"TXST": txmgrLog,
```

Comment:
Not a big deal, but probably should make the prefix match the name.  If it's going to be called tx manager now, ideally it would be referred to that throughout.  How about `"TMGR"` or `"TXMR"`?